### ISSUE TITLE
Form View Font Size Can't be Set or Changed

### ISSUE THREAD

**Issue by balajiramanathan** [pawelsalawa/sqlitestudio#3997]
### Details
Under the look and feel tab of the Configuration dialog, there are font choices for various things like SQL Editor, Data View, etc.  But the Data View font only applies to Grid View, not to Form View.  The font used in the Form View is very tiny and it is not comfortable (sometimes not even possible for me) to read the data once I switch to form view (for editing multi-line data fields, for example).

Please make the Data View font and font size applicable to Form View also, or better yet, provide a separate font style and size configuration for Form View.  Thank you.

**Comment by pawelsalawa** [comment]
Actually it is more likely that all font configurations will disappear from the application. Font sizes (or in general - entire interface size) should be controlled by the operating system, because if you need to enlarge some font, it means you probably have it larger everywhere else too. That's why operating systems have capability to scale up icons, fonts, buttons, etc.

**Comment by tuffnatty** [comment]
> Actually it is more likely that all font configurations will disappear from the application. Font sizes (or in general - entire interface size) should be controlled by the operating system, because if you need to enlarge some font, it means you probably have it larger everywhere else too. That's why operating systems have capability to scale up icons, fonts, buttons, etc.

I strongly disagree. You have scale and font face customisation in Web browsers, right? One may need different fonts for data and GUI controls. for example. To me, the font configurability is one of the strongest points of attraction in SQLiteStudio.
BTW, here on OS X I don't see any OS level option to change default font and scale.

**Comment by pawelsalawa** [comment]
I see. This will be tough.
I agree not to throw away fonts configuration (at least not that easily :) ). I need to yet run some experiments with custom fonts & UI scaling (it's there under Windows 10 for sure, I will check other systems too).

Basically this needs further investigation.

**Comment by jonathan-g** [comment]
A related issue is that I often want to specify a monospace font for the form view (e.g., suppose a data field contains source code, and I want to use a monospaced font to edit it in the form). 

I would like to be able to choose to view forms using a monospaced font.
